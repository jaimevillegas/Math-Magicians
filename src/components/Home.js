export default function Home() {
  return (
    <div title="home-wrapper" className="home-wrapper">
      <h1 className="home-title">Welcome to our page!</h1>
      <p className="home-paragraph">
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
      </p>
      <p className="home-paragraph">
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Exercitationem suscipit assumenda autem ab blanditiis
        error harum? Temporibus culpa tempora, repellat sapiente
        ullam nulla. Culpa distinctio nobis voluptatibus nostrum
        expedita soluta.
      </p>
    </div>
  );
}
