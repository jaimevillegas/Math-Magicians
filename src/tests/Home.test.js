import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

describe('Home test', () => {
  test('Home component is displayed on the HTML', () => {
    render(<Home />);
    const home = screen.getByTitle('home-wrapper');
    expect(home).toMatchSnapshot();
  });
});