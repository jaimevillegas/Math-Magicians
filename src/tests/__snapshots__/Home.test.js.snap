// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test Home component is displayed on the HTML 1`] = `
<div
  class="home-wrapper"
  title="home-wrapper"
>
  <h1
    class="home-title"
  >
    Welcome to our page!
  </h1>
  <p
    class="home-paragraph"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta.
  </p>
  <p
    class="home-paragraph"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Exercitationem suscipit assumenda autem ab blanditiis error harum? Temporibus culpa tempora, repellat sapiente ullam nulla. Culpa distinctio nobis voluptatibus nostrum expedita soluta.
  </p>
</div>
`;
